cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java

```diff
@@ -565,6 +568,43 @@ boolean isUserTaskDoneExceptionally() {
     }
   }
 
+  // Persist user request and result to log. Called when the user task completes and is moved into COMPLETED status
+  private void persistUserTask(UserTaskInfo userTaskInfo) {
+    String userTaskId = userTaskInfo.userTaskId().toString();
+    String endPoint = userTaskInfo.endPoint().toString();
+    String requestUrl = userTaskInfo.requestWithParams();
+    String startDate = KafkaCruiseControlUtils.toDateString(userTaskInfo.startMs(), DATE_FORMAT, TIME_ZONE);
+    List<OperationFuture> futures = userTaskInfo.futures();
+
+    StringBuilder message = new StringBuilder();
+    message.append(String.format("UserTask [%s] for endpoint [%s] with requestUrl [%s] started-at [%s].",
```

Comment:
looks like `requestUrl` already have `endpoint` information.

Reply:
yeah it's not as searchable because it will look like 'http://hostname:portname/kafkacruisecontrol/endpoint?params', I think it's easier for user to make it more explicit.